source/scenes/gameplay/PPGameScene.cpp

```diff
@@ -107,6 +107,17 @@ void GameScene::update(float timestep) {
     _palette->update();
     _action->update(activeCanvases, _palette->getSelectedColor());
     _globalTimer->update(_state.getLevelTimer());
+    
+    // Check if the level is complete
+    if (activeCanvases.size() == 0) {
+        this->_complete = true;
+        auto levelcomplete = PolygonNode::allocWithTexture(_assets->get<Texture>
+            ("levelcomplete"));
+        levelcomplete->setContentSize(Application::get()->getDisplaySize());
+        // levelcomplete->setContentSize(Application::get()->getDisplaySize()/1.3);
+        //levelcomplete->setAnchor(Vec2::ANCHOR_CENTER);
```

Comment:
Tried to make smaller and center, but having difficulty getting the node to center dynamically.